### PR TITLE
docs(readme): add action inputs

### PR DESCRIPTION
**Pull request by Casey-Kiewit**
Also documents previously undocumented input and clarifies some descriptions.

Resolves https://github.com/ybiquitous/npm-audit-fix-action/issues/749.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -22,11 +22,11 @@ inputs:
     description: "Default branch"
     required: false
   commit_title:
-    description: "Commit title"
+    description: "PR title"
```
Comment (ybiquitous) [inline action.yml:25]: [suggest] `commit_title` is used for a commit message title. Could you please rewrite it?

```suggestion
    description: "Commit and PR title"
```

File: README.md [hunk 2]
```diff
@@ -27,7 +27,17 @@ jobs:
...
+### Action Inputs
+
+| Key | Value |
+| --- | ----- |
+| github_token | GitHub token |
+| github_user | GitHub user name for commit changes |
+| github_email | GitHub user email for commit changes |
+| branch | Created branch |
+| default_branch | Default branch |
+| commit_title | PR title |
```
Comment (ybiquitous) [inline README.md:39]:
```suggestion
| commit_title | Commit and PR title |
```

**Review by ybiquitous — CHANGES_REQUESTED** [review]
Thanks for the pull request! Please consider my reviews.

**Comment by Casey-Kiewit** [comment]
@ybiquitous Thanks for the review! This is ready for another look whenever you have a couple minutes.

**Review by ybiquitous — APPROVED** [review]
Thank you!

**Pull request merged**